cypress/support/config/settings.js

```diff
@@ -4514,11 +4514,11 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: [],
+            paths: ['/sinhala/world-52909487'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/sinhala/sri-lanka-23037210'],
             enabled: false,
           },
           local: {
```

Comment:
Same for Sinhala?

Reply:
Added [here](https://github.com/bbc/simorgh/pull/6788/commits/4767d5abefd627065a1a1f550058ea7a25951316).